modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
+
+[IMPORTANT]
+====
+It is highly discouraged to skip this prerequisite. If the OpenShift CLI `oc` is not updated to the target version before your update, unexpected issues may occur.
+====
+
+.Procedure
+
+. Update any OLM Operators to versions that are compatible with both versions you are updating to.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
+To view the status of all machine config pools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. Change to the `eus-4.12` channel. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.12
+----
++
+[NOTE]
+====
+
+If you receive an error message indicating that `eus-4.12` is not one of the
+available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS updates.
+This rollout process generally takes 45-90 days starting at the GA date.
+====
++
+
+. Pause all worker machine pools except for the master pool. Run the following command:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+----
+
+. Update to the latest version. Run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.11.18
+----
+
+. Ensure that the updates are complete by reviewing the cluster version. Run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
```

Comment:
I realize this is in the outgoing text, so feel free to punt, but `oc adm upgrade` will display ClusterVersion status with more context, and is probably preferable.  For example, `oc get clusterversion` will not render `ReleaseAccepted=False` conditions, where the cluster-version operator refuses to accept the requested target.

Reply:
Updated both steps with clusterversion to oc adm upgrade :) good call